src/assets/scripts/client/math/flightMath.js

```diff
@@ -144,17 +126,19 @@ export function isWithinAirspace(airport, pos) {
  * @return {boolean}
  */
 export function calculateDistanceToBoundary(airport, pos) {
+    // TODO: remove perimeter
     const perim = airport.perimeter;
```

Comment:
@erikquinn Can we come up with a solution that does not require a `perimeter` property? maybe something based on the actual airspace.

Reply:
Probably? It was only intended to reduce the number of "is this aircraft within this polygon?" calculations. If we had a whole bunch of polygons that defined many shelves of airspace, we might have to do an enormous number of these calculations, each of which are probably extremely expensive. My original thought was having a perimeter would mean that anybody outside of the outermost extents of the airspace can at least only be compared against a single polygon instead of many. Perhaps this could be generated from the others mathematically at instantiation to have the same effect? Could be tough though.

We just want to make sure we're doing everything we can to keep the sim running smoothly, and determining whose airspace an aircraft is in without it taking too long and causing the whole thing to slow down.